src/Uno.CrossTargetting.props

```diff
@@ -106,7 +106,7 @@
 	</ItemGroup>
 	<MakeDir Directories="$(_TargetNugetFolder)" />
 
-	<Message Importance="high" Text="OVERRIDING NUGET PACKAGE CACHE: $(_TargetNugetFolder)" />
+	<Message Importance="high" Text="OVERRIDING NUGET PACKAGE CACHE: $(_TargetNugetFolder) and $(_TargetNugetRefFolder)" />
```

Comment:
Is this fixing the problem that xaml files are not moved to the package cache also or it's unrelated?  (if it's related I'll apply that fix to all our repos)

Reply:
It's unrelated, it's just about telling which reference folder is being updated.